mh/quest_utils.py

```diff
@@ -0,0 +1,1564 @@
+import struct
+
+
+LAYER_CHAT_COLORS = (0xbb3385ff, 0xffffffff, 0xffffffff)
+
+QUEST_FLAG_UNK = 0b1000  # It really is unknown, it put a question mark icon on the monster/item box icon
+QUEST_FLAG_ARENA = 0b10000
+QUEST_FLAG_DOUBLE_MONSTER = 0b10000
+
+
+def pad(s, size, p=b'\0'):
+    data = bytearray(s + p * max(0, size-len(s)))
+    data[-1] = 0
+    return data
+
+
+# location_type enum
+class Location_Type():
+    QUEST_LOCATION_NONE = 0
+    QUEST_LOCATION_D_ISLAND = 1
+    QUEST_LOCATION_SANDY_PLAINS = 2
+    QUEST_LOCATION_FLOODED_FOR = 3
+    QUEST_LOCATION_TUNDRA = 4
+    QUEST_LOCATION_VOLCANO = 5
+    QUEST_LOCATION_GRT_DESERT = 6
+    QUEST_LOCATION_UW_RUIN = 7
+    QUEST_LOCATION_LAND_ARENA_1 = 8
+    QUEST_LOCATION_LAND_ARENA_2 = 9
+    QUEST_LOCATION_WATER_ARENA_1 = 10
+    QUEST_LOCATION_SACRED_LAND = 11
+    QUEST_LOCATION_WATER_ARENA_2 = 12
+location_type = Location_Type()
```

Comment:
The comment can be removed and the parentheses as well. We usually don't use underscore (`_`) in class name. The class doesn't need to be instantiated, you can do it directly like that: `LocationType.QUEST_LOCATION_NONE`.

Example:
```python
class MyEnum:
    A = 42

print("The answer is {}".format(MyEnum.A))
```